src/ZODB/multicommitadapter.py

```diff
@@ -0,0 +1,65 @@
+"""Adapt non-IMultiCommitStorage storages to IMultiCommitStorage
+"""
+
+import zope.interface
+
+from .ConflictResolution import ResolvedSerial
+
+class MultiCommitAdapter:
+
+    def __init__(self, storage):
+        self._storage = storage
+        ifaces = zope.interface.providedBy(storage)
+        zope.interface.alsoProvides(self, ifaces)
+        self._resolved = set()
```

Comment:
Then, a comment on such useless lines would be good (btw, what about `None` instead of `set()`). It's horrible to ignore the purpose of a line when refactoring code.